### PR TITLE
[VEN-2872] Deploy VTreasuryV8 to Base Sepolia

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves #

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Chicken12834** [comment]
> Code Coverage
> 
> Package	Line Rate	Branch Rate	Health
> contracts	100%	100%	✔
> contracts.Admin	88%	41%	✔
> contracts.Comptroller	100%	90%	✔
> contracts.Comptroller.Diamond	95%	61%	✔
> contracts.Comptroller.Diamond.facets	76%	64%	➖
> contracts.Comptroller.Diamond.interfaces	100%	100%	✔
> contracts.DelegateBorrowers	100%	89%	✔
> contracts.Governance	68%	45%	➖
> contracts.InterestRateModels	74%	59%	➖
> contracts.Lens	45%	37%	❌
> contracts.Liquidator	83%	60%	✔
> contracts.Oracle	100%	100%	✔
> contracts.PegStability	88%	84%	✔
> contracts.Swap	88%	58%	✔
> contracts.Swap.interfaces	100%	100%	✔
> contracts.Swap.lib	81%	55%	✔
> contracts.Tokens	100%	100%	✔
> contracts.Tokens.Prime	96%	72%	✔
> contracts.Tokens.Prime.Interfaces	100%	100%	✔
> contracts.Tokens.Prime.libs	90%	76%	✔
> contracts.Tokens.VAI	81%	53%	✔
> contracts.Tokens.VRT	20%	9%	❌
> contracts.Tokens.VTokens	64%	47%	➖
> contracts.Tokens.VTokens.legacy	0%	0%	❌
> contracts.Tokens.XVS	19%	8%	❌
> contracts.Utils	50%	30%	❌
> contracts.VAIVault	50%	45%	➖
> contracts.VRTVault	49%	36%	❌
> contracts.XVSVault	64%	50%	➖
> contracts.lib	89%	71%	✔
> **Summary**	**62%** (3346 / 5369)	**46%** (1245 / 2736)	➖

**Comment by Chicken12834** [comment]
> ## Description
> Resolves #
> 
> ## Checklist
> * [ ]  I have updated the documentation to account for the changes in the code.
> * [ ]  If I added new functionality, I added tests covering it.
> * [ ]  If I fixed a bug, I added a test preventing this bug from silently reappearing again.
> * [ ]  My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

File: hardhat.config.ts [hunk 1]
```diff
@@ -207,6 +207,12 @@ const config: HardhatUserConfig = {
...
       accounts: DEPLOYER_PRIVATE_KEY ? [`0x${DEPLOYER_PRIVATE_KEY}`] : [],
     },
+    basesepolia: {
+      url: process.env.ARCHIVE_NODE_basesepolia || "https://sepolia.base.org",
+      chainId: 84532,
+      live: true,
+      accounts: DEPLOYER_PRIVATE_KEY ? [`0x${DEPLOYER_PRIVATE_KEY}`] : [],
+    },
   },
   etherscan: {
```
Comment (chechu) [inline hardhat.config.ts:217]: I think we should also add an entry in `etherescan.customChains`, for `basesepolia`

**Review by chechu — COMMENTED** [review]
I think we should also add an entry at `.env.example`, suggesting the use of `https://sepolia.base.org`

**Review by chechu — APPROVED** [review]

File: hardhat.config.ts [hunk 2]
```diff
@@ -302,6 +308,14 @@ const config: HardhatUserConfig = {
...
         },
       },
+      {
+        network: "basesepolia",
+        chainId: 84532,
+        urls: {
+          apiURL: "https://api-sepolia.basescan.org/api",
+          browserURL: "https://sepolia.basescan.org/",
+        },
+      },
```
Comment (GitGuru7) [inline hardhat.config.ts:318]: We also need to add base sepolia api key [here](https://github.com/VenusProtocol/venus-protocol/pull/529/files#diff-57cf5378f8cab20b02b279097ba6fad08eb53f747d81770045dda9c5f95effe3R228)

**Review by GitGuru7 — COMMENTED** [review]
Also we need to add RPC url in .env.example [here](https://github.com/VenusProtocol/venus-protocol/blob/develop/.env.example#L22)
as we are doing for all other networks.

File: deployments/basesepolia.json [hunk 3]
```diff
@@ -0,0 +1,200 @@
+{
+  "name": "basesepolia",
+  "chainId": "84532",
+  "contracts": {
+    "VTreasuryV8": {
+      "address": "0x07e880DaA6572829cE8ABaaf0f5323A4eFC417A6",
```
Comment (GitGuru7) [inline deployments/basesepolia.json:6]: https://sepolia.basescan.org/address/0x07e880DaA6572829cE8ABaaf0f5323A4eFC417A6#code

Contract is not verified, Can you please verify it.

**Review by chechu — APPROVED** [review]

File: deployments/basesepolia.json (hunk 3 above)
Comment (chechu) [reply deployments/basesepolia.json:6]: Done

File: hardhat.config.ts (hunk 2 above)
Comment (chechu) [reply hardhat.config.ts:319]: https://github.com/VenusProtocol/venus-protocol/pull/529/commits/03b21e4997bdd6a0e9ec08f749699c111bbc7183

**Pull request merged**